frontend/src/v4/modules/sequences/sequences.sagas.ts

```diff
@@ -179,7 +179,11 @@ function * prefetchFrames() {
 function* updateSelectedStateDefinition() {
 	const selectedFrame = yield select(selectSelectedFrame);
 	const stateDefinitions = yield select(selectStateDefinitions);
-	yield put(SequencesActions.setSelectedStateDefinition(stateDefinitions[selectedFrame?.state]));
+	const selectedStateDefinition = stateDefinitions[selectedFrame?.state];
```

Comment:
When I enter the sequence it shows everything until the frame is showed. 
![showing-everything](https://github.com/3drepo/3drepo.io/assets/42032433/099b387e-0957-4e23-b4b5-5d8efd19c20a)

Is it possible to show the same state until the frame is loaded?

Reply:
To be fixed in #4926